test/cases/parsing/optional-chaining/test.filter.js

```diff
@@ -0,0 +1,12 @@
+const supportsOptionalChaining = require("../../../helpers/supportsOptionalChaining");
+
+/**
+ * @param {import("../../../../").Configuration} config
+ * @returns {boolean}
+ */
+module.exports = function (config) {
+	if (config.mode === "production") return false;
```

Comment:
Terser does not supports optional chaining yet.
https://github.com/terser/terser/issues/567

nullish coalescing works on try.terser.org, but not optimized.